app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+	def full_title(page_title = '')
```

Comment:
1. 2 space indentation
2. Use double quote instead of using single quote
3. Remove optional parenthesis ()